packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1806,6 +1820,27 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
     _cancelActivePointers();
   }
 
+  /// Method which sends the route name changes over System Channels.
+  ///
+  /// There are two methods: 'pushRoute', 'popRoute'. 'pushRoute' is also used
```

Comment:
There are three methods define above

Reply:
Thanks looks like I totally forgot to replace this one after doing the latest changes!